src/DotVVM.Framework/Diagnostics/DotvvmDiagnosticsConfiguration.cs

```diff
@@ -9,24 +9,41 @@ namespace DotVVM.Framework.Diagnostics
 
     public class DotvvmDiagnosticsConfiguration
     {
-        public DotvvmDiagnosticsConfiguration()
+       
+
+        private DiagnosticsServerConfiguration configuration;
+        private DateTime configurationLastWriteTimeUtc;
+
+        public string DiagnosticsServerHostname
         {
-            LoadConfiguration();
+            get
+            {
+                EnsureConfigurationValid();
```

Comment:
What is the point of this change? Why not load the configuration in constructor?

Reply:
In case that Visual Studio changes the address while app is running.